src/control/server/ctl_storage_rpc.go

```diff
@@ -492,7 +492,6 @@ func (c *ControlService) StorageFormat(ctx context.Context, req *ctlpb.StorageFo
 	// TODO: supply whitelist of instance.Devs to init() on format.
 	for _, srv := range instances {
 		if instanceErrored[srv.Index()] {
-			srv.log.Errorf(msgFormatErr, srv.Index())
```

Comment:
Was this message noisy? It seems like format failure is something interesting to log

Reply:
Hmm, I was thinking that it was redundant to other errors, but I'll put it back for the moment.